### ISSUE TITLE
spatialite-gui

### ISSUE THREAD

**Issue by lucaarcteam** [archeos/ArcheOS#84]
We should package spatialite-gui (https://www.gaia-gis.it/fossil/spatialite_gui/index). A deb package is present in Debian Sid repos (http://packages.debian.org/sid/spatialite-gui).

**Comment by lucaarcteam** [comment]
Since latest version of spatialite-gui has been already packaged for Jessie, we can simply package an older version for ArcheOS Theodoric and update it in ArcheOS-next. Up to now we need this software just becaus most common open source GIS do not manage sqlite and spatialite data with all their potentalities. Maybe in the future this problem will be solved autonomously. What do you think @df79, @alearcteam ?

**Comment by lucaarcteam** [comment]
By now we could also use the Debian GIS Project repos, to add this and other software.

http://pkg-grass.alioth.debian.org/

**Comment by df79** [comment]
I agree. Spatialite-gui is not mandatory, but usefull for not SQLite and
SpatiaLite experts. I think we have to mantain this software packaging an
old version or using Debian GIS repos (or using jessie backports, if
spatialite-gui is available there.
Denis

2015-05-20 13:12 GMT+02:00 Luca Bezzi <notifications@github.com>:
